### ISSUE TITLE
Is Sentry.showReportDialog deprecated in @sentry/nextjs?

### ISSUE THREAD

**Issue by Vadorequest** [getsentry/sentry-docs#3720]
## Issue

This an issue with:

Platform/SDK documentation (Which one?)
https://docs.sentry.io/platforms/javascript/guides/nextjs/
Sentry Product (Which area?)
nextjs

## Description

I'm migrating from a mix of @sentry/node and @sentry/browser to the latest @sentry/nexjs plugin.

But one of the feature available in `@sentry/browser` doesn't seem to be available anymore (`showReportDialog`).
![image](https://user-images.githubusercontent.com/3807458/121580943-9b227d80-ca2d-11eb-8ff7-f5e0d77ec373.png)

### Suggested Solution

Add documentation about whether it has been removed and how to use it with `@sentry/nextjs`.

**Comment by chadwhitacre** [comment]
Thoughts on this one, @iker-barriocanal @lobsterkatie? Is this a valid issue that we can put on the backlog?

**Comment by AbhiPrasad** [comment]
Yes we can add it to the backlog - we are aware of the issue and are working on a fix, should have updated the issue, thanks for the reminder @chadwhitacre

**Comment by chadwhitacre** [comment]
Thanks @AbhiPrasad ... doing the label dance. :)

**Comment by ankur99** [comment]
any update on this issue?????
